modules/admin-ui-frontend/app/scripts/shared/services/restServiceMonitor.js

```diff
@@ -45,6 +48,42 @@ angular.module('adminNg.services')
 
     Monitoring.getActiveMQStats();
     Monitoring.getBasicServiceStats();
+    Monitoring.getVersionStats();
+  };
+
+  Monitoring.getVersionStats = function() {
+    $http.get(MY_VERSION_PATH).then(function(response_my_version) {
+      $http.get(LATEST_VERSION_PATH).then(function(response_latest_version) {
+        Monitoring.populateService(LATEST_VERSION_NAME);
+
+        if (response_latest_version.status === 200 && response_my_version.status === 200
+        && response_my_version.data.consistent) {
+          var my_version = response_my_version.data.version;
+          var latest_version = response_latest_version.data;
+          services.service[LATEST_VERSION_NAME].docs_url =
+            'https://docs.opencast.org/r/' + latest_version[0] + '.x/admin/';
+
+          if (parseFloat(my_version) >= parseFloat(latest_version) || my_version.endsWith('SNAPSHOT')) {
+            services.service[LATEST_VERSION_NAME].status = OK;
+            services.service[LATEST_VERSION_NAME].error = false;
+          } else if (my_version[0] == latest_version[0]) {
+            services.service[LATEST_VERSION_NAME].status = 'There is a minor update available.';
+            services.service[LATEST_VERSION_NAME].error = true;
+          } else if (parseInt(latest_version[0]) - parseInt(my_version[0]) <= 2) {
```

Comment:
```suggestion
          } else if (parseInt(latest_version[0]) - parseInt(my_version[0]) < 2) {
```

Currently, 8.6 is the latest version and 8.x and 7.x is supported.
If someone has a 6.4 then `8 - 6 = 2` which should yield the `…is no longer supported. Please update.`